shell/platform/android/io/flutter/view/FlutterView.java

```diff
@@ -557,7 +559,7 @@ public boolean onTouchEvent(MotionEvent event) {
             }
         }
 
-        assert packet.position() % (kPointerDataFieldCount * kBytePerField) == 0;
+        assert packet.position() % (kPointerDataFieldCount * kPointerBytesPerField) == 0;
```

Comment:
Just filed https://github.com/flutter/flutter/issues/25517

Reply:
Ahh, OK, I'll fix that too in this PR then.